docs/application/native/tutorials/details/io-overview.md

```diff
@@ -26,7 +26,7 @@ The following table shows the application directory hierarchy:
 | `data`           | Used to store private data of an application.<br>To get the application data directory path, call the `app_get_data_path()` function of the App Common module. | Owner: Read and Write<br>Others: Access denied |
 | `res`            | Used to read resource files that are delivered with the application package.<br>To get the application resource directory path, call the `app_get_resource_path()` function of the App common module. | Owner: Read<br>Others: Access denied     |
 | `shared`         | Parent directory of the data, res, and trusted sub-directories. Files in this directory cannot be delivered with the application package. | Owner: Read<br>Others: Read              |
-| `shared/data`    | Used to share data with other applications.<br>To get this directory path of your own application, call the `app_get_shared_data_path()` function of the App common module.<br>Files stored in the shared/data directory can be read by other applications. Do not store application's private data in the shared/data directory. | Owner: Read and Write<br>Others: Read    |
+| `shared/data`    | Used to share data with other applications.<br>To get the directory path of your application, call the `app_get_shared_data_path()` function of the App common module.<br>Files stored in the shared/data directory can be read by other applications. Do not store application's private data in the shared/data directory. Since Tizen 3.0, an application that wants to use shared/data directory must declare http://tizen.org/privilege/appdir.shareddata privilege. If the application does not declare the privilege, the framework will not create shared/data directory for the application. | Owner: Read and Write<br>Others: Read    |
```

Comment:
How about changing to `http://tizen.org/privilege/appdir.shareddata`? It can confuse user as kind of a URL.

Reply:
I think the link may be useful because the link contains description about the http://tizen.org/privilege/appdir.shareddata.

Reply:
In the future, www.tizen.org site is planned to shut down and replaced. So the additional link to the www.tizen.org is not recommended.
Also, as I explained, `http://tizen.org/privilege/appdir.shareddata` is actually not a URL. The page is created to prevent claim from the users who misunderstood privileges & features as broken URls.
So, would you please apply code style to privilege?

Reply:
I've updated the patch. However, some URL like string in tizen privilege and features are managed to be linked to valid pages. I think you need to consider carefully when you change the address of www.tizen.org.

Reply:
Thanks for understanding & updating. Also, we will take care of the links to www.tizen.org.